src/components/ArchivedReportFooter.tsx

```diff
@@ -53,14 +53,35 @@ function ArchivedReportFooter({report, reportClosedAction, personalDetails = {}}
         policyName = lodashEscape(policyName);
     }
 
-    const text = shouldRenderHTML
-        ? translate(`reportArchiveReasons.${archiveReason}`, {
-              displayName: `<strong>${displayName}</strong>`,
-              oldDisplayName: `<strong>${oldDisplayName}</strong>`,
-              policyName: `<strong>${policyName}</strong>`,
-              shouldUseYou: actorPersonalDetails?.accountID === getCurrentUserAccountID(),
-          })
-        : translate(`reportArchiveReasons.${archiveReason}`);
+    let text;
```

Comment:
`text` is now of type any 😢

Reply:
Let's add  `: string` here, thanks!

Reply:
@shubham1206agra @blazejkustra Alternatively, we can directly set the text value within the switch case as like below:

```javascript
const text =(()=> {
    switch (archiveReason) {
        case CONST.REPORT.ARCHIVE_REASON.ACCOUNT_CLOSED:
            return translate(`reportArchiveReasons.${archiveReason}`, {
                displayName: `<strong>${displayName}</strong>`,
            });
        case CONST.REPORT.ARCHIVE_REASON.ACCOUNT_MERGED:
            return translate(`reportArchiveReasons.${archiveReason}`, {
                displayName: `<strong>${displayName}</strong>`,
                oldDisplayName: `<strong>${oldDisplayName}</strong>`,
            });
        case CONST.REPORT.ARCHIVE_REASON.REMOVED_FROM_POLICY:
            return translate(`reportArchiveReasons.${archiveReason}`, {
                displayName: `<strong>${displayName}</strong>`,
                policyName: `<strong>${policyName}</strong>`,
                shouldUseYou: actorPersonalDetails?.accountID === getCurrentUserAccountID(),
            });
        case CONST.REPORT.ARCHIVE_REASON.POLICY_DELETED:
            return translate(`reportArchiveReasons.${archiveReason}`, {
                policyName: `<strong>${policyName}</strong>`,
            });
        default:
            return translate(`reportArchiveReasons.${archiveReason}`);
    }
})()
```

Reply:
@shubham1206agra  why use let here? I don't see any logic in it since we aren't going to reassign the text.